byzanz-gui

```diff
@@ -39,19 +35,31 @@ fi
  
 # Window geometry
 XWININFO=$(xwininfo)
+
+read TITLE < <(grep -o '".*"'  <<< "$XWININFO")
+TITLE=$(echo $TITLE | sed 's/"//g')
 read X < <(awk -F: '/Absolute upper-left X/{print $2}' <<< "$XWININFO")
 read Y < <(awk -F: '/Absolute upper-left Y/{print $2}' <<< "$XWININFO")
 read W < <(awk -F: '/Width/{print $2}' <<< "$XWININFO")
 read H < <(awk -F: '/Height/{print $2}' <<< "$XWININFO")
  
 # Notify the user of recording time and delay
-notify-send "GIFRecorder" "Recording duration set to $D seconds. Recording will start in $DELAY seconds."
+echo "Recording duration set to $D seconds. Recording will start in $DELAY seconds."
  
 #Actual recording
+espeak "Starting in $DELAY seconds."
 sleep $DELAY
-beep
-byzanz-record -c --verbose --delay=0 --duration=$D --x=$X --y=$Y --width=$W --height=$H "$FOLDER/GIFrecord_$TIME.gif"
-beep
+espeak "Start"
+
+if [ $TITLE = "unity-panel" ]; then
```

Comment:
Is there a way we could check for DE/WM == unity, in addition to the window title?

Could you please add a note about this feature to the README? Thanks!